src/Common/Core/Impl/Idle/IdleTimeAction.cs

```diff
@@ -4,10 +4,8 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.Common.Core.Shell;
-using Microsoft.Languages.Core.Utility;
-using Microsoft.Languages.Editor.Shell;
 
-namespace Microsoft.Languages.Editor.Tasks {
+namespace Microsoft.Common.Core.Idle {
```

Comment:
I'm not fond of moving this class to the `Common`. It is not just some idle, it is a very specific event that happens only in application with UI thread when there are no scheduled tasks and there is no user interaction for some period of time. It doesn't have any meaning in Broker or in the out-of-proc parts of the application.

Why do you need it in Common?